.ci/integration_test.groovy

```diff
@@ -90,12 +90,13 @@ def call(ghprbActualCommit, ghprbCommentBody, ghprbPullId, ghprbPullTitle, ghprb
                         sh """
                         cp -R /home/jenkins/git/tispark/. ./
                         git checkout -f ${ghprbActualCommit}
-                        find core/src -name '*Suite*' > test
+                        find core/src -name '*Suite*' | grep -v 'MultiColumnPKDataTypeSuite' > test
+                        shuf test -o  test2
+                        mv test2 test
+                        find core/src -name '*MultiColumnPKDataTypeSuite*' >> test
```

Comment:
why add `MultiColumnPKDataTypeSuite` to the end?

Reply:
We use round robin to split all tests evenly (especially `MultiColumnPKDataTypeSuite`, ordering them together ensures that 36 suites are separated evenly in 18 test containers), also they run longer and should be executed later to minimize test waiting period.